pkg/controller/operators/catalog/operator.go

```diff
@@ -132,12 +137,14 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		client:                 crClient,
 		lister:                 lister,
 		namespace:              operatorNamespace,
-		resolver:               resolver.NewOperatorsV1alpha1Resolver(lister, crClient),
+		resolver:               resolver.NewOperatorsV1alpha1Resolver(lister, crClient, opClient.KubernetesInterface(), operatorNamespace),
```

Comment:
note to self: why is this needed?

Reply:
It's used for launching the bundle image (LaunchBundleImage).